dbschema/progress-report.variance-explanation.esdl

```diff
@@ -0,0 +1,12 @@
+module ProgressReport {
+  type VarianceExplanation extending ProgressReport::Child, Engagement::Child {
+    required report: default::ProgressReport {
+      readonly := true;
+    };
```

Comment:
Already declared by ancestor, but we need to an exclusive constraint.
```suggestion
    overloaded report {
      constraint exclusive;
    };
```
Since there's a 1-1 relationship between ProgressReports and VarianceExplanation.